src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineWindowsLauncher.java

```diff
@@ -225,6 +240,7 @@ public boolean verifyServerHostKey(
         // keep retrying until SSH comes up
         logInfo(computer, listener, "Failed to connect via ssh: " + e.getMessage());
         logInfo(computer, listener, "Waiting for SSH to come up. Sleeping 5.");
+        e.printStackTrace();
```

Comment:
You can log the stacktrace: https://docs.oracle.com/javase/6/docs/api/java/util/logging/Logger.html#log(java.util.logging.Level,%20java.lang.String,%20java.lang.Throwable)

Reply:
Just realized that I left this in from debugging the integration tests, so thanks for catching this. Good to know for the future though